Sources/NIOCore/AsyncChannel/AsyncChannelHandler.swift

```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-import DequeModule
+@preconcurrency import DequeModule
```

Comment:
Do we actually need a `@preconcurrency` import? `Deque` is `Sendable` if its `Element` is `Sendable`.

Reply:
Ah, because we support versions of swift-collections so old that they don't have that conformance. Let's lift the constraint.

Reply:
Now fixed.